### PR TITLE
Add offline flag + add optional JS download in HTML Pass

### PR DESCRIPTION
**Pull request by dbMundada**

### CODE REVIEW

**Review by pandurangpatil — APPROVED** [review]
looks good

File: src/main/scala/ai/privado/passes/HTMLParserPass.scala [hunk 1]
```diff
@@ -92,6 +94,16 @@ class HTMLParserPass(cpg: Cpg, projectRoot: String, ruleCache: RuleCache) extend
       options.setThrowExceptionOnFailingStatusCode(false) // Do not throw exceptions for failing HTTP status codes
       options.setThrowExceptionOnScriptError(false)       // Do not throw exceptions for JavaScript errors
 
+      // Disable loading of CSS and images
+      options.setCssEnabled(false)
+      options.setDownloadImages(false)
+      options.setPrintContentOnFailingStatusCode(false)
+
+      // Based on Offline mode skip downloading JS files
+      if (privadoInputConfig.offlineMode) {
```
Comment (nikvin15) [inline src/main/scala/ai/privado/passes/HTMLParserPass.scala:103]: Awesome 🚀

**Comment by khemrajrathore** [comment]
@dbMundada Need to make similar change in janus and privado-cli

**Comment by nikvin15** [comment]
> @dbMundada Need to make similar change in janus and privado-cli

Once the janus changes are done let me know so that I can make the changes in andromeda to use this flag when running it in offline mode.

**Review by khemrajrathore — APPROVED** [review]
LGTM

**Pull request merged**